### DISCUSSION TITLE
include examples

### DISCUSSION THREAD

**Discussion by nagordon** [DLR-AE/LoadsKernel#7] [Questions & Answers] [answered]
Hello, 
   This is really impressive work and I am excited to try it. I would like to try an demo to understand the software better, but there are no examples included. Could you add some to the repo?

The documentation shows a link to some examples, but it is not accessible. 
https://gitlab.dlr.de/loads-kernel/loads-kernel-examples

Thanks

**Comment by ArneVoss** [comment]
Hello Neal, 
thank you for your interest in our software! Currently, I am still in the process of moving not only software, but also everything around it towards open source. However, I can't publish the examples as they are taken from DLR internal projects. This will change within the next months with the help of a student who will prepare a dedicated example as part of his master thesis. 

In the meantime: If you already have a structural model in Nastran (some experience in aeroelasticty would be a plus), we can have a video call and talk about your project.

**Reply by ArneVoss** [reply, marked as answer]
Hey, 
I just wanted to let you know that there are now a number of examples and tutorials available based on the Douglas DC-3 aircraft.
